src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -68,19 +108,23 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
         };
     }
 
-    @computed public get onSelectDataSet() {
+    @bind
+    public onSelectProfileX(option:any) {
         if (this.props.store.molecularProfileIdToMolecularProfile.isComplete) {
```

Comment:
theres no longer a need to wait for this promise to be ready - you're not using it's result on the inside of the if statement